packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener-rule.ts

```diff
@@ -60,7 +60,7 @@ export interface BaseApplicationListenerRuleProps {
    *
    * @default - No path condition.
    */
-  readonly pathPattern?: string;
+  readonly pathPattern?: string | string[];
```

Comment:
No union types please. The fact that they are currently supported by jsii is a mistake and we will take them out eventually.

Add a new field named `pathPatterns?: string[]` and add validation to make sure they're not both set.

Reply:
I'm still seeing the old commits. Did you push them?